android/settings.gradle

```diff
@@ -55,12 +53,6 @@ project(':react-native-community-netinfo').projectDir = new File(rootProject.pro
 include ':react-native-screen-brightness'
 project(':react-native-screen-brightness').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-screen-brightness/android')
 include ':app'
-includeBuild('../node_modules/react-native-gradle-plugin')
-if (settings.hasProperty("newArchEnabled") && settings.newArchEnabled == "true") {
```

Comment:
We still have `newArchEnabled` in `android/gradle.properties`.  Is this still useful, or we can remove it?

Reply:
yes, they all take the info by the properties file